src/components/web-monitoring-ui.jsx

```diff
@@ -77,8 +77,9 @@ export default class WebMonitoringUi extends React.Component {
    * Load all pages.
    * @private
    *
-   * Sends a requests out to localApi for pages and sets `pages` property of state.
+   * Sends a request to fetch the first chunk of pages and sets the `pages` property of state.
    * These are passed as props to various child components.
+   * If the user is not logged in, throws an error.
```

Comment:
This isn’t quite right; instead of throwing, we set the `pages` property of the state to the error instance instead of an array of `Page` objects.